templates/text-embeddings/README.ipynb

```diff
@@ -346,7 +346,13 @@
    "source": [
     "### Scaling with GPUs\n",
     "\n",
-    "To use GPUs for inference in the Workspace, we can specify `num_gpus` and `concurrency` in the `ds.map_batches()` call below to indicate the number of embedding models and the number of GPUs per model instance, respectively. For example, with `concurrency=4` and `num_gpus=1`, we have 4 embedding model instances, each using 1 GPU, so we need 4 GPUs total."
+    "To use GPUs for inference in the Workspace, we can specify `num_gpus` and `concurrency` in the `ds.map_batches()` call below to indicate the number of embedding models and the number of GPUs per model instance, respectively. For example, with `concurrency=4` and `num_gpus=1`, we have 4 embedding model instances, each using 1 GPU, so we need 4 GPUs total.\n",
+    "\n",
+    "You can also specify which accelerator type (for example, if your model requires a specific GPU) to use with the `accelerator_type` parameter. For a full list of supported accelerator types, see [the documentation](https://docs.ray.io/en/latest/ray-core/accelerator-types.html).\n",
+    "\n",
+    "If you do not need to select a particular instance type, you can omit this parameter and select \"Auto-select worker nodes\" under the compute configuration to have Ray select the best worker nodes from the available types:\n",
+    "\n",
+    "![title](assets/ray-data-gpu.png)"
```

Comment:
Hmm. This image should only highlight the "Auto-select worker nodes" checkbox? We should not highlight the name of the worker node below? I don't know which to look at.